README.md

```diff
@@ -36,6 +36,8 @@ To try `pgo` simply modify `config/example.config` by replacing the `host`, `dat
 `default` is the name of the pool, `size` is the number of connections to create for the pool. Or you can start the pool through `pgo:start_pool/2` which creates it as a child of `pgo`'s simple one for one:
 
 ``` erlang
+> application:ensure_all_started(pgo).
+{ok,[]}
```

Comment:
```suggestion
{ok,[backoff,opentelemetry_api,pg_types,pgo]}
```

Reply:
Nice suggestion [0ba2862](https://github.com/erleans/pgo/pull/73/commits/0ba28628bc9a9e3854153f0e3a4373b5307d547f). I like how this shows what is being started when you run this command. I had already started these which is why the list is empty I guess. Thanks Tristan!